website/src/components/Pills/index.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+import { useState } from "react";
+
+export default function Pills({
+  options,
+  multiselect,
+}: {
+  options: Array<string>;
+  multiselect: boolean;
+}) {
+  const [filters, setFilters] = useState("All Posts");
+
+  return (
+    <div className="flex flex-wrap justify-center md:justify-start md:flex-row gap-3">
+      {options.map((option, index) => (
+        <button
+          key={index}
+          className={`flex px-8 py-4 rounded-full leading-3 border-[1px] hover:brightness-95 ${

```

Comment:
```suggestion
          className={`flex px-8 py-4 rounded-full leading-3 border-[1px] hover:brightness-95 ${

```

Hmm, why use `px` here and not just `border-1`? It might be preferable to avoid hardcoding pixel values here in case we need to scale the `em` / `rem` size right?